app/api/events.py

```diff
@@ -31,18 +31,63 @@ class Meta:
 
     id = fields.Str(dump_only=True)
     identifier = fields.Str(dump_only=True)
-    name = fields.Str()
-    event_url = fields.Url()
+    name = fields.Str(required=True)
+    email = fields.Str(required=True)
+    event_url = fields.Url(dump_only=True)
```

Comment:
Change this back to fields.Str(),there are some invalid values in the database right now, they willbe fixed in another issue